lib/action_cable/connection/base.rb

```diff
@@ -12,16 +12,16 @@ module Connection
     #   module ApplicationCable
     #     class Connection < ActionCable::Connection::Base
     #       identified_by :current_user
-    #   
-    #       def connect
+    #
+    #       after_connect do
```

Comment:
Ah, I hadn't seen that there were only `after` callbacks. Then it makes even less sense to get rid of the methods for me.